BUILD.bazel

```diff
@@ -27,3 +27,113 @@ java_binary(
     main_class = "com.bazel_diff.Main",
     runtime_deps = ["@bazel_diff//jar"],
 )
+
+alias(
+    name = "envoy_controller",
```

Comment:
Nit: Is there any kind of ordering here? Perhaps we could sort them alphabetically?

Reply:
It is in the order the bazel query returns the results:
`bazel query "attr(tags, service, //...)"`